src/index.js

```diff
@@ -57,36 +57,32 @@ export default function (validators = []) {
 
 				if (validator) {
 					const {valid, messages} = await validator.validate(record);
+					const response = {
+						description: validator.description,
+						state: 'invalid',
+						messages
+					};
+
+					if (typeof (response.messages) === 'undefined') {
+						delete response.messages;
+					}
```

Comment:
Use a function to generate the response. Include message only if it's defined instead of deleting it afterwards.